service.go

```diff
@@ -65,6 +65,7 @@ func (s *Service) AsyncReadings() bool {
 
 // Start the device service.
 func (s *Service) Start(errChan chan error) (err error) {
+	start := time.Now()
```

Comment:
we should create a `startTime` variable in Service struct, and record `time.Now()` at the beginning of `NewService` func because `configLoader.LoadConfig` is executed in `NewService` func

Reply:
Just fixed it.